Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,338 @@
+    /// The data model that helps determine the view.
+    @StateObject private var model = Model()
+    
+    /// Manages the presentation state of the menu.
+    @State private var isPresented: Bool = false
+    
+    var body: some View {
+        MapViewReader { proxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: model.graphicOverlays
+            )
+            .onSingleTapGesture { _, tapPoint in
+                model.centerPoint = tapPoint
+            }
+            .task(id: model.centerPoint) {
+                guard let centerPoint = model.centerPoint else { return }
+                await proxy.setViewpoint(
+                    Viewpoint(center: centerPoint, scale: 1e7)
+                )
+            }
+            .toolbar {
+                // The menu which holds the options that change the ellipse and sector.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Settings") {
+                        isPresented = true
+                    }
+                    .disabled(model.centerPoint == nil)
+                    .popover(isPresented: $isPresented) {
+                        Form {
+                            ParameterSlider(
+                                label: "Axis Direction:",
+                                value: $model.axisDirection,
+                                range: 0...360
+                            )
+                            .onChange(of: model.axisDirection) {
+                                model.refreshSector()
+                            }
+                            
+                            Stepper(
+                                value: $model.maxPointCount,
+                                in: 0...1000,
+                                step: 1
+                            ) {
+                                Text("Max Point Count:  \(String(format: "%d", model.maxPointCount))")
+                            }
+                            .font(.caption)
+                            .onChange(of: model.maxPointCount) {
+                                model.refreshSector()
+                            }
+                            
+                            ParameterSlider(
+                                label: "Max Segment Length:",
+                                value: $model.maxSegmentLength,
+                                range: 1...1000
+                            )
+                            .onChange(of: model.maxSegmentLength) {
+                                model.refreshSector()
+                            }
+                            
+                            GeometryTypeMenu(
+                                selected: $model.selectedGeometryType
+                            )
+                            .onChange(of: model.selectedGeometryType) {
+                                model.refreshSector()
+                            }
```

Comment:
I'm afraid that you've reinvented the wheel with `GeometryTypeMenu`. That same kind of interface has already been provided by SwiftUI's `Picker`:

```swift
Picker("Geometry Type", selection: $model.selectedGeometryType) {
    ForEach(GeometryType.allCases, id: \.self) { geometryType in
        Text(geometryType.label)
    }
}
.onChange(of: model.selectedGeometryType) {
    model.refreshSector()
}
```

Reply:
Yeah, this was created with keeping all the settings visible at the same time with the map, but it's outlived its purpose.